### PR TITLE
Provide a way to send sms's using local numbers

### PR DESCRIPTION
**Pull request by cadecairos**

### CODE REVIEW

**Comment by cadecairos** [comment]
@jbuck R?

File: workers/send_sms.js [hunk 1]
```diff
@@ -20,10 +20,18 @@ module.exports = function(sid, authToken, from) {
       return cb(new Error("Missing required `body` param"));
     }
 
+    var sendFrom = from,
+          toCountryCode = data.to.split(" ")[0].replace("+", ""),
```
Comment (cadecairos) [inline workers/send_sms.js]: note: incoming phone numbers will all have the format `+<num><space><rest of number>`

**Comment by jbuck** [comment]
Could you add some more context to this bug? What do you mean by 'local number'?

**Comment by cadecairos** [comment]
Basically, if I have a spanish number, and sawmill is configured to do so, it will send an sms from a spanish number.

**Pull request closed without merging**